changelogs/fragments/dnf-update-only-latest.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - update only - packages were failing when update-only was set as well as latest was selected (https://github.com/ansible/ansible/issues/81376).
```

The following should describe the problem better and mention the module name as well:
```suggestion
  -  dnf - fix a failure when a package from URI was specified and ``update_only`` was set (https://github.com/ansible/ansible/issues/81376).
```